docs/en/04_Changelogs/4.9.0.md

```diff
@@ -40,6 +41,16 @@ SilverStripe\Assets\Image:
 
 CMS authors can disable lazy loading on images added via the WYSIWYG editor in the "Edit image" panel by setting the new "Loading" field to "Eager".
 
+### Session manager added to silverstripe/recipe-cms {#session-manager}
```

Comment:
This is a headline feature ... seems like we should have something punchy that sells it a bit more.

```suggestion
### Manage your CMS sessions across devices{#session-manager}
```

Reply:
Updated